<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="83" started-at="2019-11-20T22:02:07Z" finished-at="2019-11-20T22:02:07Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="83" started-at="2019-11-20T22:02:07Z" finished-at="2019-11-20T22:02:07Z">
      <class name="practiceTests.BalancedBinaryTreeTests">
        <test-method status="PASS" signature="balancedBTTests_1()[pri:0, instance:practiceTests.BalancedBinaryTreeTests@7c0e2abd]" name="balancedBTTests_1" duration-ms="35" started-at="2019-11-20T14:02:07Z" finished-at="2019-11-20T14:02:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- balancedBTTests_1 -->
        <test-method status="FAIL" signature="balancedBTTests_2()[pri:0, instance:practiceTests.BalancedBinaryTreeTests@7c0e2abd]" name="balancedBTTests_2" duration-ms="7" started-at="2019-11-20T14:02:07Z" finished-at="2019-11-20T14:02:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at practiceTests.BalancedBinaryTreeTests.balancedBTTests_2(BalancedBinaryTreeTests.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- balancedBTTests_2 -->
      </class> <!-- practiceTests.BalancedBinaryTreeTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
